cypress.json

```diff
@@ -8,6 +8,6 @@
   "supportFile": "tests/e2e/support/index.ts",
   "screenshotOnRunFailure": false,
   "video": false,
-  "testFiles": "**/*.features",
+  "testFiles": "**/*.feature",
```

Comment:
After updating to Webpack v5 via `vue-cli` dependencies using the `features` option no longer works. This makes the CI slower to up to 2 minutes. Given that this is the archetype and not the main project is somewhat assumable, but we'll have to keep an eye on it and try to find a solution in the future.

Reply:
<img src="https://media0.giphy.com/media/in6mnJNYjGKpq/giphy.gif"/>